### DISCUSSION TITLE
Python 3.8

### DISCUSSION THREAD

**Discussion by rawrke** [duckdb/duckdb#3626] [Q&A] [answered]
Are there plans in the near-term to support `DuckDB` on Python 3.8+?

**Comment by Mytherin** [comment, marked as answer]
We support and build releases for Python up to and including Python 3.10. Are you running into any problems there?

**Reply by rawrke** [reply]
Yeah I am just getting started with `duckdb`, so I may be missing something. 

I have started with a blank conda environment with Python 3.9, and when I install duckdb with `conda install -c conda-forge python-duckdb`. It downgrades my Python version to `3.7.12`.

And looking at your [**`Python.yml`**](https://github.com/duckdb/duckdb/blob/master/.github/workflows/Python.yml) for your CI I noticed the `python-version` was locked in at 3.7. This made me assume that you are only supporting up to 3.7.

**Reply by Mytherin** [reply]
[This is the relevant line in the CI](https://github.com/duckdb/duckdb/blob/master/.github/workflows/Python.yml#L176) - 3.7, 3.8, 3.9 and 3.10 are all supported. I'm not sure about the exact problem with Conda, however. Perhaps somebody else can help you there.

**Reply by Alex-Monahan** [reply]
I'm not sure on the conda side,but since DuckDB only depends on numpy, I would recommend just pip installing it! The dependency tracking of conda isn't super necessary for DuckDB.

**Reply by rawrke** [reply]
@Alex-Monahan,  That did seem to work, I made a new env and just pip installed it. Got it in an environment with Python 3.8.

Thanks for the tip and quick feedback @Alex-Monahan @Mytherin
